app/models/friend.rb

```diff
@@ -0,0 +1,13 @@
+class Friend < ApplicationRecord
+  belongs_to :user
+  belongs_to :friended, class_name: 'User'
+
+  after_create :create_inverse
+
+  def create_inverse
```

Comment:
I like this method so we will create an entry into our self joins table for the inverse of the friendship relationship